conda_forge_tick/new_update_versions.py

```diff
@@ -70,35 +73,34 @@ def new_update_upstream_versions(
 
     # Inspection the graph object and node update:
     # print(f"Number of nodes: {len(gx.nodes)}")
-    Node_count = 0
-
+    node_count = 0
     to_update = []
     for node, node_attrs in tqdm.tqdm(_all_nodes):
-        attrs = node_attrs["payload"]
-        if attrs.get("bad") or attrs.get("archived"):
-            attrs["new_version"] = False
-            continue
-        to_update.append((node, attrs))
+        with node_attrs["payload"] as attrs:
+            if attrs.get("bad") or attrs.get("archived"):
+                # "new_version" = False
```

Comment:
please remove unused code

Reply:
I leave it there to serve as a comment for what's happening, but it's better to remove

Reply:
If you want to leave a descriptive comment then I would use prose rather than commented out code.